kolibri/utils/cli.py

```diff
@@ -579,12 +579,18 @@ def status():
 
 
 @main.command(cls=KolibriDjangoCommand, help="Start worker processes")
+@click.option(
+    "--port",
+    default=OPTIONS["Deployment"]["HTTP_PORT"],
+    type=int,
+    help="Port on which to run Kolibri services",
```

Comment:
Note that this is incorrect - this should be `Port on which Kolibri is being served` - it's an informative setting, not one that is forcing behaviour :)

Reply:
I see, so the port is not a prescription

Reply:
It's prescriptive when it is passed to the `kolibri start` command, but `kolibri services`, does not run the server itself, and so can't set the port.